frontend/src/components/Layout/Header/SearchCourses/SearchResultItem.tsx

```diff
@@ -14,23 +14,23 @@ import { Image, CourseTitle, SearchResultCourse } from './styled';
 interface CourseFound {
   course: Course;
   addDivider: boolean;
-  foundInMyCourses: string | undefined;
-  status: string | undefined;
+  foundInMyCoursesId?: string;
+  status?: string;
 }
 
 const LABEL_MESSAGE = 'New';
 
 const SearchResultItem: React.FC<CourseFound> = ({
   course,
   status,
-  foundInMyCourses,
+  foundInMyCoursesId,
   addDivider,
 }) => (
   <Link
     key={course._id}
     to={
-      foundInMyCourses
-        ? `${PATHS.myCourses}/${foundInMyCourses}`
+      foundInMyCoursesId
+        ? `${PATHS.myCourses}/${foundInMyCoursesId}`
         : `${PATHS.coursesList}/${course._id}`
     }
```

Comment:
Let's wait for Daniel PR to be merged and rebase on develop: those lines will be changed

Reply:
done